src/components/SettingsGrid/SettingsGrid.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import React, { PureComponent, Children } from 'react'
+
+type Props = {|
+  +children: Array<React$Node>,
+|}
+
+class SettingsGrid extends PureComponent<Props, *> {
+  render() {
+    const { children } = this.props
+    const { map: childrenMap } = Children
+
+    return (
+      <ul className='j-grid'>
+        {childrenMap(children, (card, index) => (
```

Comment:
why not just `Children.map(...`?

Reply:
better to check that children is array with `typeof`
and iterate over it - `children.map`, otherwise `null`